Dockerfile

```diff
@@ -1,19 +1,31 @@
 FROM python:3.8
 
-WORKDIR /
+ARG UID
+ARG GID
+
+# set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
```

Comment:
I'm assuming this is for the logging as non-root user?

Reply:
No, setting these two environment variables is not related to logging as non-root user. From what I read, it is good practice to set these two environment variables in a docker container environment, so I added them, which should not have any side effect, but let me know if you disagree. I can remove them if there could be any side effect.